removefiles/activity.go

```diff
@@ -0,0 +1,51 @@
+package removefiles
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+)
+
+const ActivityName = "remove-files-activity"
+
+type Activity struct {
+	logger logr.Logger
+}
+
+func NewActivity(logger logr.Logger) *Activity {
+	return &Activity{logger: logger}
+}
+
+type ActivityParams struct {
+	// RemoveNames is the comma separated list of filenames that should be
+	// removed from Path.
+	RemoveNames string
+
+	// Path is the directory from which files should be removed.
+	Path string
+}
+
+type ActivityResult struct {
+	// Count is the number of files removed from Path.
+	Count int
+}
+
+// Execute deletes any files named by RemoveNames from the given Path and
+// returns a count of deleted files.
+func (a *Activity) Execute(
+	ctx context.Context,
+	params *ActivityParams,
+) (*ActivityResult, error) {
```

Comment:
Does golines fix this the other way around? If they fit in 120 would it transform this 6 lines into two?

Reply:
No. As far as I know golines only breaks long lines, I've never seen it join short lines.

Reply:
That's sad ;(